config/config.js

```diff
@@ -16,6 +16,9 @@ module.exports = {
   cookiebot: {
     scriptUrl: process.env.COOKIEBOT_SCRIPT_URL || '//assets.nhs.uk/scripts/cookie-consent.js',
   },
+  headerFooterApi: {
+    endpoint: process.HEADER_FOOTER_API_ENDPOINT || 'https://nhsuk-live.nhswebsite-integration.nhs.uk/content-api/header-footer.json',
```

Comment:
This should be process.**env.**HEADER_FOOTER_API_ENDPOINT right? 
Does this mean that if the env var is missing it defaults to the integration URL? If so, when running in the live environment could this potentially end up pointing to the integration endpoint?